terraform/README.md

```diff
@@ -68,16 +70,16 @@ aws_secret_access_key=FYd4t...
 File: ~/.aws/config
 
 ```
-[profile dm-main-account-infrastructure-preview]
-source_profile=dm-andras
-mfa_serial=arn:aws:iam::<main account id>:mfa/<your username>
-role_arn = arn:aws:iam::<main account id>:role/infrastructure
+[profile development-infrastructure]
+source_profile=default
+mfa_serial=arn:aws:iam::<main account id>:mfa/<IAM username>
+role_arn=arn:aws:iam::<digitalmarketplace-development account id>:role/infrastructure
```

Comment:
Aha, interesting - this might be why I haven't been able to run this stuff successfully!

Reply:
Might be better to link to this (https://alphagov.github.io/digitalmarketplace-manual/aws-accounts.html#available-roles) rather than provide a partial extract.

Maybe.